### PR TITLE
Add touch events of platform view channel for tizen

### PR DESCRIPTION
**Pull request by seungsoo47**
* Send a event type, a button type and platform view's positions to
  a platform view which is registered for platformview factory

### CODE REVIEW

File: shell/platform/tizen/channels/platform_view_channel.cc [hunk 1]
```diff
@@ -126,8 +137,27 @@ void PlatformViewChannel::HandleMethodCall(
...
+          return;
+        }
+        type = std::get<int>(event[0]);
+        button = std::get<int>(event[1]);
+        x = std::get<double>(event[2]);
+        y = std::get<double>(event[3]);
+        dx = std::get<double>(event[4]);
+        dy = std::get<double>(event[5]);
+
+        // LoggerD("Engine::Touch type[%s],btn[%d],x[%f],y[%f],dx[%f],dy[%f]",
```
Comment (bbrto21) [inline shell/platform/tizen/channels/platform_view_channel.cc:155]: Please remove commented codes

Comment (seungsoo47) [reply shell/platform/tizen/channels/platform_view_channel.cc:155]: Do we have to remove all logs? I think this log may be used later.

Comment (swift-kim) [reply shell/platform/tizen/channels/platform_view_channel.cc:155]: Commented out code should be removed in production code when there's no special reason.

Comment (swift-kim) [reply shell/platform/tizen/channels/platform_view_channel.cc:155]: You can create a development branch if the feature is still under development.

Comment (seungsoo47) [reply shell/platform/tizen/channels/platform_view_channel.cc]: Fixed it.

**Review by bbrto21 — APPROVED** [review]
LGTM

**Review by bwikbs — APPROVED** [review]
LGTM

**Pull request merged**